app/constants/navbar.js

```diff
@@ -1,2 +0,0 @@
-export const LOGOUT_URL = 'https://api.realdevsquad.com/auth/signout';
```

Comment:
In `constants/signup.js` there are two more URLs present, please put them into `url.js` as well

Reply:
Resolved